lib/specifications/projects/views.js

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
This is kind of a pain to rename... how do you feel about calling these project views and renaming the views folder to `templates`?